bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/UpdateEspIdfMasterPropertyTester.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright 2022 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+package com.espressif.idf.ui;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.eclipse.core.expressions.PropertyTester;
+import org.eclipse.jgit.api.Git;
+
+import com.espressif.idf.core.logging.Logger;
+import com.espressif.idf.core.util.IDFUtil;
+
+public class UpdateEspIdfMasterPropertyTester extends PropertyTester
+{
+	@Override
+	public boolean test(Object receiver, String property, Object[] args, Object expectedValue)
+	{
+		// Check if the master version of ESP-IDF is being used. If false, do not show update command
+		String branchName = ""; //$NON-NLS-1$
```

Comment:
We can use StringUtil.EMPTY here for initializing empty strings

Reply:
added